thunder/core/dtypes.py

```diff
@@ -242,6 +268,8 @@ def __init__(self, name, shortname, *, bytes, is_weak):
 
 float_dtypes = {d for d in all_dtypes if isinstance(d, floating)} | {float}
 
+float_8bit_dtypes = {d for d in all_dtypes if (isinstance(d, inexact) and d.bytes <= 1)}
```

Comment:
Is `isinstance(d, inexact)` required instead of `isinstance(d, floating)` (which is used in the above line for `float_dtypes`)? Also, I think we should keep `d.bytes = 1`, as in future we may have dtypes smaller than a byte.

Reply:
Agreed; `inexact` includes the complex types, which probably doesn't matter practically, bit it's nice to be more consistent with the construction of `float_dtypes` by querying `floating`